modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/adapters/registry/CallrailAdapterSpec.scala

```diff
@@ -34,6 +34,8 @@ class CallrailAdapterSpec extends Specification with DataTables with ValidatedMa
   toRawEvents should return a Validation Failure if there are no parameters on the querystring      $e2
   """
 
+  val AdapterWithDefaultConfig = new CallrailAdapter(config = callrailConfig)
```

Comment:
```suggestion
  val adapterWithDefaultConfig = new CallrailAdapter(config = callrailConfig)
```
We generally use lowerCamelCase in the codebase

Reply:
Changed.